lbrytv/middleware/redirect.js

```diff
@@ -13,6 +13,11 @@ async function redirectMiddleware(ctx, next) {
     return;
   }
 
+  if (path === '/') {
```

Comment:
conditional by user auth state until new homepage is done? this won't make much sense for unauth users

Reply:
That's harder to do unless we call internal-apis with the auth token. I figured we could still keep the url (even if it's a little confusing) but still show the signed out view